metricbeat/module/traefik/health/_meta/testdata/docs.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
There is a file like this one in `_meta/test/simple.json`, and there are similar files in other modules, not sure how they were generated, but should we use them instead? It can be weird to have both `test` and `testdata` directories with test files.

Reply:
I did not use the files in `test` in other directories on purpose to not have to deal with potentially other tests in this PR. In case we get decide to move forward with this approach, we should decide afterwards which existing tests should be removed. I kept all existing tests for now to make this PR independent. 

In the end I expect only `testdata` to be there but having both during a migration period.